xrally_kubernetes/tasks/scenarios/services.py

```diff
@@ -93,7 +93,7 @@ def run(self, image, port, protocol, command=None, custom_endpoint=False,
         self.client.create_job(
             name=name,
             namespace=namespace,
-            image="appropriate/curl",
+            image="appropriate/curl:edge",
```

Comment:
I'm ok to merge it now, but should it be configurable? (via scenario argument or config file?)